DevelopedCalculator.js

```diff
@@ -1,5 +1,7 @@
 class DevelopedCalculator extends Calculator {
     history = [this.currentResult];
+    historyDiv = null;
+    historyDiv = document.querySelector('div');
```

Comment:
`document.querySelector` returns the first match element. in this case it searching in the DOM and returning the first `div` element.

are you sure the history div is the first div element in the page and are you sure that it will be always the first div element in page?

also, I though you're creating the history element in the `render` method

Reply:
When I create it inside the method i get error message

Reply:
> When I create it inside the method i get error message
creatining what?